models/passenger.js

```diff
@@ -0,0 +1,101 @@
+const { db } = require('../database/connection.js')
+
+const getPassengerIdByName = (name) => {
+  if(typeof name !== 'string') {
+    return 'Please enter a passenger name.'
+  }
```

Comment:
From my past feedback from SEPs, we should handle errors by throwing an error. I see the usefulness of your approach and have used it in the past.